js/webui/src/mediasession_controller.js

```diff
@@ -0,0 +1,78 @@
+import DataSource from "./data_source";
+import PlayerModel from "./player_model";
+import silenceMp3 from "./5-seconds-of-silence.mp3";
+
+export default class MediaSessionController {

```

Comment:
Formatting does not match other files

Reply:
Changed to single quotes and tab width 4